src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/reporting/model/BookingsReportRow.kt

```diff
@@ -3,6 +3,15 @@ package uk.gov.justice.digital.hmpps.approvedpremisesapi.reporting.model
 import java.time.LocalDate
 
 data class BookingsReportRow(
+  val referralId: String?,
+  val referralDate: LocalDate?,
+  val riskOfSeriousHarm: String?,
+  val sexOffender: Boolean?,
+  val needForAccessibleProperty: Boolean?,
+  val historyOfArsonOffence: Boolean?,
+  val dutyToReferMade: Boolean?,
+  val dateDutyToReferMade: LocalDate?,
+  val isReferralEligibleForCas3: Boolean?,
```

Comment:
`referralEligibility` is now ready to add so we could add it here or as another new PR?

Reply:
I've added this column as `referralEligibilityReason`.